parity-util-mem/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 repository = "https://github.com/paritytech/parity-common"
 description = "Collection of memory related utilities"
 license = "GPL-3.0"
+edition = "2018"
```

Comment:
don't we need `[lib] proc-macro = true` in order make the procedural macro `MallocSizeOf` to work? 

Otherwise, I think we need to `#[macro_use] extern crate parity_util_mem as malloc_size_of;` which is annoying IMHO.

Reply:
I just tested with 'proc-macro = true', it is only for procedural crate, here we reexport procedural crate, so it was a no success for me :( . I will therefore approve the pr as his.

Reply:
Ok, I see thanks @cheme :P